src/darwin/Framework/CHIP/MTROTAImageTransferHandler.mm

```diff
@@ -310,7 +310,9 @@ - (instancetype)initWithMTROTAImageTransferHandler:(MTROTAImageTransferHandler *
 
     TransferSession::OutputEventType eventType = event.EventType;
 
-    ChipLogError(BDX, "OutputEvent type: %s", event.ToString(eventType));
+    if (eventType != TransferSession::OutputEventType::kNone) {
+        ChipLogError(BDX, "OutputEvent type: %s", event.ToString(eventType));
```

Comment:
Why is this an error log?  This should be a progress log, I would think.  Yes, I know, pre-existing...  Followup for changing that, maybe.

Reply:
Will fix.

Reply:
fixed.